docs/_documentations/eclipse-getting-started.md

```diff
@@ -19,7 +19,8 @@ To install Codewind for Eclipse, complete the following steps:
     - Install Eclipse IDE Version 2019-09 R (4.13.0) or later to avoid [Bug 541220](https://bugs.eclipse.org/bugs/show_bug.cgi?id=541220).
     - **Note:** the earliest supported version of the Eclipse IDE is Version 2019-03 (4.11).
 2. Install [Docker](https://docs.docker.com/install/) 17.06 or later. If you use Linux, you must also install [Docker Compose](https://docs.docker.com/compose/install/).
-3. Install [Codewind from Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
+3. If you are planning to work with Appsody projects, enable your local disk drive for sharing in Docker. To do this, open the **Docker Settings** window, click the **Shared Drives** tab, and select the drive on which you are installing Codewind. 
```

Comment:
`local disk drive` sounds very old fashioned

Reply:
Googled, local drive is more popular so updated to that.